manuscript/events.md

```diff
@@ -254,7 +254,7 @@ else {
     event = document.createEvent('CustomEvent');
     event.initCustomEvent('image-removed', false, true,
       {id: 1}
-    });
+    );
```

Comment:
This is a good catch, but I think what I'll do is move the starting bracket up to the end of line 255, and then keep the original ending bracket in place. This makes it more consistent with the other custom event example used later on (line 529).